lib/rules/strict-component-boundaries.js

```diff
@@ -17,19 +17,22 @@ module.exports = {
       ImportDeclaration(node) {
         const importSource = node.source.value;
         const resolvedSource = resolve(importSource, context);
-        const pathParts = resolvedSource
-          ? pathSegmantsFromSource(resolvedSource)
-          : pathSegmantsFromSource(importSource);
+        const pathParts = pathSegmantsFromSource(importSource);
```

Comment:
In my opinion, this is not the most useful way to fix this issue. I would rather you never reference the image source, and instead operate on the relative source from the root of the app. Otherwise, I don’t think you’ll ever be able to get this to actually reflect what we want to enforce